src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid.Tests/app.config

```diff
@@ -1,8 +0,0 @@
-﻿<runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Do the tests not need this any longer?

Reply:
It didn't have the root tag ```<configuration>``` so it was resulting in an error.

Reply:
OK if the tests are passing don't worry about this.